### PR TITLE
Modify doc generation after repositories merge

### LINKED ISSUE #748: BUG-Doc does not build after repositories merge
**Bug description**

Change the generation process.

**Expected change**

The doc generates again.

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Resolves #748.

### CODE REVIEW

File: tools/_setup_generation/step_gui_ext_refman.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def enter(self, setup: Setup):
     def setup(self, setup: Setup) -> None:
         if self.npm_path:
             saved_cwd = os.getcwd()
-            gui_path = os.path.join(setup.root_dir, "gui")
+            gui_path = os.path.join(setup.root_dir, "taipy-fe")
             os.chdir(gui_path)
             print(f"... Installing node modules...", flush=True)
             subprocess.run(f"{self.npm_path} run inst", shell=True)
```
Comment (FredLL-Avaiga) [inline tools/_setup_generation/step_gui_ext_refman.py]: could change "run inst" to "i" I think

File: tools/fetch_source_files.py [hunk 2]
```diff
@@ -229,26 +223,33 @@ def copy(item: str, src: str, dst: str, rel_path: str):
                 dest_path = os.path.join(ROOT_DIR, "taipy")
                 if not os.path.exists(dest_path):
                     os.makedirs(dest_path)
-                src_path = os.path.join(src_path, "src", "taipy")
-                for item in os.listdir(src_path):
-                    copy(item, src_path, dest_path, "")
+                sources_path = os.path.join(src_path, "taipy")
```
Comment (FredLL-Avaiga) [inline tools/fetch_source_files.py:226]: evaluate the opportunity of using pathlib.Path for clarity ?
- Path(ROOT_DIR) / "taipy"
- Path(...).exists()
- Path(...).iterdir() ...

**Review by FredLL-Avaiga — APPROVED** [review]

File: tools/fetch_source_files.py (hunk 2 above)
Comment (FabienLelaquais) [reply tools/fetch_source_files.py:226]: Will do. Just not now. Whatever works. Thx

File: tools/_setup_generation/step_gui_ext_refman.py (hunk 1 above)
Comment (FabienLelaquais) [reply tools/_setup_generation/step_gui_ext_refman.py]: I'll try that

Comment (FabienLelaquais) [reply tools/_setup_generation/step_gui_ext_refman.py:43]: And... `npm ERR! Missing script: "i"`
Let me track back...
Thanks

Comment (FabienLelaquais) [reply tools/_setup_generation/step_gui_ext_refman.py]: Yep, that works. Thanks.

File: docs/contributing/contributing.md [hunk 3]
```diff
@@ -53,14 +53,17 @@ sprint is attached to the `current sprint`. Please, do not work on it, the Taipy
...
-Taipy is organized in several repositories:
-
-- [taipy-config](https://github.com/Avaiga/taipy-config).
-- [taipy-core](https://github.com/Avaiga/taipy-core).
-- [taipy-gui](https://github.com/Avaiga/taipy-gui).
-- [taipy-rest](https://github.com/Avaiga/taipy-rest).
-- [taipy-templates](https://github.com/Avaiga/taipy-templates).
-- [taipy](https://github.com/Avaiga/taipy) brings previous packages in a single one.
+The Taipy source code is located in the is organized [taipy](https://github.com/Avaiga/taipy)
+repository, in the `taipy` directory.
```
Comment (jrobinAV) [inline docs/contributing/contributing.md:57]: The Taipy source code is located in the [taipy](https://github.com/Avaiga/taipy)
repository, in the `taipy` directory.

**Review by jrobinAV — COMMENTED** [review]
Small "cocquille" in the contributing.md

File: docs/contributing/contributing.md (hunk 3 above)
Comment (FabienLelaquais) [reply docs/contributing/contributing.md:57]: Thanks. Next merge will fix that.

**Pull request merged**